Src/Service/Service/service/ProjectService.cs

```diff
@@ -40,24 +40,24 @@ public void AddProject(IProject project)
             UnitOfWork.Projects.Add(project);
         }
 
-        public IProjectApplication[] GetAllApplications(IProject project)
+        public IProjectApplication[] GetAllApplications(int projectId)

```

Comment:
I'm going to leave this as an int (but change the name). My thinking is this should save us a DB call.